src/dpsvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/GetValidOpenDFCMCase.java

```diff
@@ -11,7 +15,10 @@
     @RequestMapping(value = "/getValidOpenDFCMCase",
             produces = {"application/xml"},
             method = RequestMethod.GET)
-    public String getValidOpenDFCMCase(@RequestParam(value = "driversLicense", required = true) String driversLicense, @RequestParam(value = "surcode", required = true) String surcode) {
+    @ApiOperation(value = "Generates application callback endpoints for DPS Validation Service", notes = "", tags = {"DPSValidationService"})
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Successful operation")})
+    public String getValidOpenDFCMCase(@ApiParam(value = "driversLicense", required = true) @RequestParam(value = "driversLicense", required = true) String driversLicense, @ApiParam(value = "surcode", required = true) @RequestParam(value = "surcode", required = true) String surcode) {
         /* TODO implement the logic as in the webmethods */
         String integer_ = "2"; // This will be changed later to accept the int value from the ORDS
```

Comment:
I know this wasn't part of your PR, can you slap a TODO at the beginning of the comment on this line please :)